cirq-google/cirq_google/devices/serializable_device.py

```diff
@@ -108,6 +108,25 @@ def __init__(
         """
         self.qubits = qubits
         self.gate_definitions = gate_definitions
+        self._metadata = cirq.GridDeviceMetadata(
```

Comment:
My understanding of `DeviceMetadata` was that a given `Device` type would have metadata matching its type, or one of its parent types if the `Device` had no additional metadata of its own.

This seems to conflict with that - either `SerializableDevice` needs to subclass `GridDevice`, or its metadata needs to be the basic `DeviceMetadata`.

Reply:
Had a quick chat offline and I think we are on the same page now with this one. Going to mark as resolved.